### ISSUE TITLE
Ideas of UI

### ISSUE THREAD

**Issue by ocean0609** [Hong-Kong-Districts-Info/dashboard-hkdistrictcouncillors#74]
Welcome Page idea.
![1  Welcome Page](https://user-images.githubusercontent.com/67370580/91650361-36ef8800-eab1-11ea-8e14-86ac7bc7a5de.png)

Change clickable options for "Terms of Use" to have a smoother experience for users.
![2  Terms of Use](https://user-images.githubusercontent.com/67370580/91650364-3a830f00-eab1-11ea-831b-b15e1b5dfeef.png)

Layout idea for "Overview of a DC"
![3  Overview of a DC](https://user-images.githubusercontent.com/67370580/91650365-3ce56900-eab1-11ea-8d70-0410ee11b2a3.png)

Tip Boxes adjustment idea
![4  Small Adjustments (Tipboxes)](https://user-images.githubusercontent.com/67370580/91650380-87ff7c00-eab1-11ea-9949-f69047ac8c06.png)

**Comment by ocean0609** [comment]
#73 

for Tip Boxes' translation

**Comment by avisionh** [comment]
These are great ideas. I'll respond to each in turn, let me know what you think!

1. The Welcome Page is interesting. I can see it will be a nice intro but given that users are likely to be accessing the website via their mobiles, would erecting a welcome page mean they have to make one more extra click to get to the information they actually want?
    + I think the current way in which they can immediate get to the *Overview of a DC* tab means we are helping them get to the page they would want as fast as possible. 
1. Absolutely right about clickable buttons, it's actually more accessible because there is more space to click on. I'll get round to implementing that. 
1. Great point about moving some of the information into the DC name box. I've done that now, moving their address in there so we have less of the large infoboxes clogging up the screen and moved the phone + email details together into the same infobox. This makes everything fit much nicer on a mobile.
    + I've also removed the link to their FB page because as you said, this is incorporated in the FB feed feature. Instead, the box re-directs to the webpage @martinctc created for each DC.
    + On adding the display picture, district balls and map icon, I think @martinctc will be in a better position to implement those so will defer to him.
1. For the tip-boxes idea, unfortunately, I don't think I have much scope to easily move it. 😢  I'll keep it in mind but might prove very difficult.
    + Could you elaborate a bit more on **no place to input name?** Not sure if I'm following. 🙈

**Comment by ocean0609** [comment]
Ah i see what you did.
Cause tip 2 said "type the name of your constituency in here." , i thought there will be a individual box for inputing the constituency name 
 
<img width="1440" alt="Screenshot 2020-09-01 at 21 24 41" src="https://user-images.githubusercontent.com/67370580/91856874-b5a71980-ec99-11ea-8881-31dd6e13aeb6.png">

**Comment by ocean0609** [comment]
For 1, i think it is a bit "unfair" for the DCs, since everyone has to look A-01's page before going to their constituency.
Getting a page before the "Overview" maybe better? And not sure if we can get data from whose page is the most visited page?

The welcome page design is just for brainstorm, we might add the "Terms of Use"  there instead of a pop-up.

And i forgot to mention, we can add a "Search box" in the sidebar.

![5  Search box](https://user-images.githubusercontent.com/67370580/91859146-ab3a4f00-ec9c-11ea-826b-467a14f10338.png)

**Comment by avisionh** [comment]
> For 1, i think it is a bit "unfair" for the DCs, since everyone has to look A-01's page before going to their constituency.
> Getting a page before the "Overview" maybe better? And not sure if we can get data from whose page is the most visited page?
> 
> The welcome page design is just for brainstorm, we might add the "Terms of Use" there instead of a pop-up.
> 
> And i forgot to mention, we can add a "Search box" in the sidebar.
> 
> ![5 Search box](https://user-images.githubusercontent.com/67370580/91859146-ab3a4f00-ec9c-11ea-826b-467a14f10338.png)

Yeah, it's a good point about it being unfair for the DCs that everyone has to look at A-01's page. I think moving towards having the search bar in the sidebar will alleviate this issue, potentially. We're also thinking of having a tab dedicated to search, where the person can use the map to search, and then they get taken to the *Overview of a DC* tab. The Trello card is [here](https://trello.com/c/jgrhysut).
